packages/f/fast_io/xmake.lua

```diff
@@ -0,0 +1,21 @@
+package("fast_io")
+    set_kind("library", {headeronly = true})
+    set_homepage("https://github.com/cppfastio/fast_io")
+    set_description("Significantly faster input/output for C++20")
+    set_license("MIT")
+
+    add_urls("https://github.com/cppfastio/fast_io.git")
+    add_urls("https://bitbucket.org/ejsvifq_mabmip/fast_io.git")
+    add_urls("https://gitee.com/qabeowjbtkwb/fast_io.git")
+
```

Comment:
```
add_versions("2023.1.28", "b99b32ab429eb6256fd8de1e17fe38e4c54eb49c")
```

Reply:
But the library has multiple commits in a day.

Reply:
Having a baseline commit that you know works as the default is recommended, assuming the commit hash warugi provided builds. Users can still opt for the latest commits by specifying the main/master branch, like `add_requires("fast_io master")`

Reply:
just use latest commit. if you want to always use HEAD commit, you can set `add_requires("fast_io master")` in project xmake.lua

Reply:
We must specify a commit to ensure that it has been fully tested on the ci. Otherwise the package is unreliable.